packages/frontend/app/utils/string-encoding.ts

```diff
@@ -22,6 +22,18 @@ export async function convertObjectToQRCodeDataURL(object: any): Promise<string>
   return await QRCode.toDataURL(string);
 }
 
+export function convertObjectToUint8Array(object: any): Uint8Array {
```

Comment:
Try using a generic type instead of any here and in the next method ;)

Reply:
Addressed in 95a7735

Reply:
Realized this caused some type check errors. If we are passing a json that at its base in an array of things, how would we go about making that generic? We can't really declare an object structure because ultimately it is an array containing a few arrays and a few objects.

Reply:
Fixed typing errors, however that caused tests to fail. Going to look into why that is the case.